pkg/controllers/subnetset/subnetset_controller.go

```diff
@@ -79,6 +80,24 @@ func (r *SubnetSetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			}
 			log.V(1).Info("added finalizer on subnetset CR", "subnetset", req.NamespacedName)
 		}
+
+		// update subnetset tags if labels of namespace changed
+		vpcSubnetSets := r.Service.SubnetStore.GetByIndex(servicecommon.TagScopeSubnetCRType, subnet.SubnetTypeSubnetSet)
```

Comment:
this will retrieve all NSX subnets created for SubnetSet CR, right?  We just need to get those NSX subnets created for SubnetSet CR in this NS.

Reply:
rename "vpcSubnetSets" to something else like "nsxSubnets", since this is to retrieve NSX subnets.

Reply:
Yes, but but it only support GetByIndex by one index instead of TagScopeSubnetCRType and namespace togother.

Reply:
renamed vpcSubnetSets  to nsxSubnets